css/screen2.css

```diff
@@ -0,0 +1,379 @@
+body{
```

Comment:
give meaningful names for the css files instead of screen2 and screen3.
also certain styles can be reused in multiple css files, you can place them in  a common css file.